src/js/gi/utils/helpers.js

```diff
@@ -7,3 +7,7 @@ export function getCurrency(value) {
 
   return isNumeric(amount) ? +amount : 0;
 }
+
+export function formatCurrency(value) {
+  return `$${Math.round(value)}`;
```

Comment:
Wanted to use `Intl.NumberFormat` here, but couldn't get it to work. And didn't want to pull in a full `react-intl` package just for this.